.github/workflows/quba-e2e.yaml

```diff
@@ -0,0 +1,108 @@
+name: k8s-e2e
+
+on: [push]
+
+jobs:
+  e2e:
+    name: e2e-test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Save the kernel version
+        run: echo "KERNEL_VERSION=$(uname -r)" >> $GITHUB_ENV
+
+      - name: Build the kernel module
+        run: make KERNEL_SRC_DIR="/usr/src/linux-headers-${KERNEL_VERSION}"
+        working-directory: ci/sro-kmod
+
+      - name: Install curl
+        run: |
+          sudo apt -y update
+          sudo apt -y install curl
+
+      - name: Download and install minikube
+        run: |
+          curl -LO https://storage.googleapis.com/minikube/releases/latest/minikube_latest_amd64.deb
+          sudo dpkg -i ./minikube_latest_amd64.deb
+
+      - name: Start minikube and wait until CoreDNS is available
+        run: |
+          minikube start --driver=docker
+          kubectl wait --for=condition=available deployment coredns -n kube-system
+
+      - name: Build sro-kmod
+        run: minikube image build -t sro-kmod:local --build-opt build-arg=KERNEL_VERSION=${KERNEL_VERSION} ci/sro-kmod
+
+      - name: Build SRO
+        run: minikube image build -t sro:local .
+
+      - name: Install kubectl
+        uses: azure/setup-kubectl@v2.0
+
+      - name: Install NFD
```

Comment:
why do we need NFD installation? We don't apply an NFD CR after that, if i am not mistaken, and daemonset does not rely on full kernel version for deployment,

Reply:
The flow was developed against a version of SRO that still had a hard dependency on NFD. Trying without now.
BTW, that command installs both the operator and a default CR.

Reply:
@yevgeny-shnaidman in fact, the dependency on NFD was only dropped [downstream](https://github.com/openshift/special-resource-operator/pull/100). It is still required upstream; SRO keeps failing until it sees the NFD label that contains the kernel version.
I added again the step that installs NFD to the flow.